mrblib/env.rb

```diff
@@ -13,4 +15,16 @@ def delete(key)
     self[key] = nil
     old
   end
+
+  def fetch(key, default = NONE, &block)
+    if key?(key)
+      self[key]
+    elsif block
+      block.call(key)
+    elsif default != NONE
+      default
+    else
+      raise KeyError, "Key not found: #{key}"
```

Comment:
Like this? https://github.com/iij/mruby-env/pull/12/commits/ef6f4e1c609b50f800f2b24feb6b4c3842ac483d